tools/cldr-code/src/main/java/org/unicode/cldr/util/VoterReportStatus.java

```diff
@@ -1,19 +1,39 @@
 package org.unicode.cldr.util;
 
 import java.util.EnumSet;
+import java.util.Set;
 
 /**
  * This interface is for objects which can expose information on which reports have been completed.
+ * @param T the type for each voter. Must be an Integer.
```

Comment:
"Must be an Integer" -- then why generic?

Reply:
See the note. It was originally generic because it didn't need to be more specific. But voteresolver's user Id is fixed as an int. I was trying to minimize churn by not removing the generic just yet.

Reply:
@btangmu  i added a note to https://unicode-org.atlassian.net/browse/CLDR-15765 to also remove the generic when that ticket goes in (post close).